tests/release/pipelines/fbc_release.go

```diff
@@ -263,7 +268,7 @@ func createFBCReleasePlanAdmission(fbcRPAName string, managedFw framework.Framew
 		Resolver: "git",
 		Params: []tektonutils.Param{
 			{Name: "url", Value: releasecommon.RelSvcCatalogURL},
-			{Name: "revision", Value: releasecommon.RelSvcCatalogRevision},
+			{Name: "revision", Value: "main"},
```

Comment:
We can't hardcode this or it won't work for testing changes to the catalog repo

Reply:
I can change it to "releaseConst.RelSvcCatalogRevision", but the case will fail.  Can we keep it as "main" until the [bug](https://issues.redhat.com/plugins/servlet/samlsso?redirectTo=%2Fbrowse%2FRHTAPREL-787) is fixed.

Reply:
You can leave it like this for testing, but I wouldn't merge it with it being changed to main. The test is not useful to us if it is using `main` for the revision

Reply:
I updated it and I'd like another PR for testing. Thanks